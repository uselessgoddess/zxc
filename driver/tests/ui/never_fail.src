#[start]
fn main(argc: isize, _: isize) -> isize {
    let x = return 12.isize;
    return x + 12.isize;
}
