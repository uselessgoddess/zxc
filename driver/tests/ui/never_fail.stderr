[E0228] Error: mismatch types
   ╭─[never_fail:1:6]
   │
 3 │     return x + 12.isize;
   │            ┬   ────┬───  
   │            ╰───────────── expected `!` 
   │                    │     
   │                    ╰───── found `isize` 
───╯
